config/rubocop.yml

```diff
@@ -12,6 +12,7 @@ AllCops:
     - 'client/**/*'
     - 'spec/dummy/db/schema.rb'
     - '**/node_modules/**/*'
+  NewCops: enable
```

Comment:
This has been the default behavior so far. If you don't add it, there will now be a warning about the new cops you need to explicitly enable or disable. Adding this, however, produces a million errors, when updating rubocop. So... not sure what's better, really. Alternatively, only checking the changes in the current branch with the new task and ignoring the rest is an option.